Lib/tracemalloc.py

```diff
@@ -280,7 +287,10 @@ def size(self):
 
     @property
     def traceback(self):
-        return Traceback(self._trace[2])
+        # Python < 3.9
+        if len(self._trace) == 3:
+            return Traceback(self._trace[2], None)
```

Comment:
This code path becomes deadcode with "traces = tuple(trace + (None,) for trace in traces)" below, no? I guess that you can now remove this code.

Reply:
You're right! I forgot we added that.